astronomer/providers/apache/hive/hooks/hive.py

```diff
@@ -0,0 +1,27 @@
+"""This module contains the Apache HiveCli hook async."""
+from airflow.hooks.base import BaseHook
+from impala.dbapi import connect
+from impala.hiveserver2 import HiveServer2Connection
+
+
+class HiveCliHookAsync(BaseHook):
+    """
+    Wrapper to interact with the Hive using impyla library
```

Comment:
This is an async hook and is not really a wrapper. Can you make it something similar as shown below:
`
    """
    Hook for Apache Livy through the REST API using LivyHookAsync

    :param livy_conn_id: reference to a pre-defined Livy Connection.
    :param extra_options: Additional option can be passed when creating a request.
            For example, ``run(json=obj)`` is passed as ``aiohttp.ClientSession().get(json=obj)``
    :param extra_headers: A dictionary of headers passed to the HTTP request to livy.

    .. see also::
        For more details refer to the Apache Livy API reference:
        https://livy.apache.org/docs/latest/rest-api.html
    """
`

Reply:
yes moved auth_mechanism to init and changed the docstring also